website/src/theme/theme.ts

```diff
@@ -78,6 +78,11 @@ const theme = createTheme({
         boxShadow: "0px 11px 33px rgba(255, 220, 179, 0.62)",
         width: "212px",
       },
+      containedSecondary: {
```

Comment:
what is the backgroundColor when not Hover for `containedSecondary`

Reply:
It is `#F26D00` i.e., secondary color in our custom material UI theme. Based on this value, material UI calculates a darker color on hover, which we are overriding here